app/jobs/job_helpers/s3_helper.rb

```diff
@@ -25,5 +25,17 @@ def s3_client
         compute_checksums: false,
       )
     end
+
+    def attempts_bucket_enabled
+      (attempts_bucket_name && attempts_bucket_name != 'default-placeholder')
+    end
+
+    def attempts_bucket_name
+      IdentityConfig.store.irs_attempt_api_bucket_name
+    end
+
+    def attempts_s3_enabled
+      IdentityConfig.store.irs_attempt_api_aws_s3_enabled && attempts_bucket_enabled
+    end
```

Comment:
I think a purist would argue that these don't belong in the S3Helper class since they're Attempts specific. But given that this was a really sparse class, and we're using this in two places, I agree with your decision to put it here rather than create something like an AttemptsS3BucketHelper class.

Reply:
Yeah - and we were already using the s3Helper for something else.  If others complain, we can definitely extract these out.